data/conf/infrasim.yml

```diff
@@ -0,0 +1,60 @@
+---
+#This file is used as InfraSIM configuration file
+
+main:
+    #Supported vnode names:

```

Comment:
Same as above, I would like to set up a dedicate session to discuss guides and documentations.